includes/components/class-listing.php

```diff
@@ -61,6 +61,9 @@ public function __construct( $args = [] ) {
 			// Alter meta boxes.
 			add_filter( 'hivepress/v1/meta_boxes/listing_settings', [ $this, 'alter_listing_settings_meta_box' ] );
 			add_filter( 'hivepress/v1/meta_boxes/listing_images', [ $this, 'alter_listing_images_meta_box' ] );
+
+            // Alter block.
```

Comment:
Please always create a pull request for changes, basic things like formatting issues shouldn't be passed further into the code review. I have a warning in WPCS here related to spaces instead of tabs. Also issues related to things like sanitization/escaping were also passed recently, these shouldn't pass the initial code review.